tests/test_adaptive_runge_kutta_scheme.py

```diff
@@ -0,0 +1,138 @@
+from typing import Callable, Tuple
+
+import numpy as np
+import pytest
+
+from rkopenmdao.runge_kutta_scheme import RungeKuttaScheme
+from rkopenmdao.butcher_tableaux import (
+    embedded_heun_euler,
+    embedded_runge_kutta_fehlberg,
+)
+from rkopenmdao.error_controllers import PID
+
+class RkFunctionProvider:
```

Comment:
You don't have to copy that stuff, you could also import it from `test_runge_kutta_scheme.py`. Or move the tests for the adaptive scheme into that file, that is also ok.